From cd2854ccfbe4debaefc3b40bd475cb4f09238c8e Mon Sep 17 00:00:00 2001
From: Leandro Ribeiro <leandro.ribeiro@collabora.com>
Date: Thu, 21 Sep 2023 16:34:25 -0300
Subject: [PATCH 504/834] loader: rename loader_open_render_node() to
 loader_open_render_node_platform_device()

This function ignore devices that are not on the platform bus. So rename
it to better reflect that.

Signed-off-by: Leandro Ribeiro <leandro.ribeiro@collabora.com>
Reviewed-by: Simon Ser <contact@emersion.fr>
Reviewed-by: Daniel Stone <daniels@collabora.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/24825>
---
 src/gallium/drivers/tegra/tegra_screen.c | 2 +-
 src/loader/loader.c                      | 2 +-
 src/loader/loader.h                      | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/gallium/drivers/tegra/tegra_screen.c b/src/gallium/drivers/tegra/tegra_screen.c
index 008aa20fb73..98bf4cd710d 100644
--- a/src/gallium/drivers/tegra/tegra_screen.c
+++ b/src/gallium/drivers/tegra/tegra_screen.c
@@ -579,7 +579,7 @@ tegra_screen_create(int fd)
 
    screen->fd = fd;
 
-   screen->gpu_fd = loader_open_render_node("nouveau");
+   screen->gpu_fd = loader_open_render_node_platform_device("nouveau");
    if (screen->gpu_fd < 0) {
       if (errno != ENOENT)
          fprintf(stderr, "failed to open GPU device: %s\n", strerror(errno));
diff --git a/src/loader/loader.c b/src/loader/loader.c
index f340b61433d..643fd5bb63e 100644
--- a/src/loader/loader.c
+++ b/src/loader/loader.c
@@ -133,7 +133,7 @@ iris_predicate(int fd)
 }
 
 int
-loader_open_render_node(const char *name)
+loader_open_render_node_platform_device(const char *name)
 {
    drmDevicePtr devices[MAX_DRM_DEVICES], device;
    int i, num_devices, fd = -1;
diff --git a/src/loader/loader.h b/src/loader/loader.h
index 2480b62d4f3..8e8845ba2fd 100644
--- a/src/loader/loader.h
+++ b/src/loader/loader.h
@@ -44,7 +44,7 @@ int
 loader_open_device(const char *);
 
 int
-loader_open_render_node(const char *name);
+loader_open_render_node_platform_device(const char *name);
 
 char *
 loader_get_render_node(dev_t device);
-- 
2.42.0

